entries/entry01.md

```diff
@@ -1,8 +1,45 @@
-# Entry 1
-##### X/X/XX
+# Entry 01
+##### 10/20/19
+---
+
+## Basic Background
+
+This is the beginning of a year-long project called the **Freedom Project**.
+We are given complete freedom :
+- our choice of design
+- our choice of partnering
+- our choice of programming language(s)
+- our choice of application (website, app, online game etc)
+
+hence the name "Freedom Project".
+
+---
+## Research Tools
+
+The very first things to do for this project were to figure out **what to make and what tools to use**.
+For example,
+- If you want **an iOS app**, you should probably spend time looking at **Swift**.
+- If you want **android app**, you should look at **Java**.
+- If you want **3D games**, then you should check out **Unity**.
+
+---
+## Why **_Swift_** ?
+
+Originally, I wanted to build my own AI bot because I believe [**Halite**](https://halite.io) was interesting for someone who enjoys math.
+However, I am also interested in making a game that brings joy, and Halite competition can be something I do on my own rather than
+putting it to be the focus of my project.
+
+As an iphone user, I would like to explore Swift and build a game into an iOS app. This means researching about Swift and working
```

Comment:
I _strongly_ encourage you to partner with a classmate and build something together. A game sounds fun, but I think you and another person or two would benefit greatly from building a more useful app. Put it out there on Slack!